src/cmd/plumb/fsys.c

```diff
@@ -199,7 +199,10 @@ startfsys(void)
 	if(post9pservice(p[1], "plumb", nil) < 0)
 		sysfatal("post9pservice plumb: %r");
 	close(p[1]);
-	proccreate(fsysproc, nil, Stack);
+	if (foreground)
```

Comment:
Generally, no spaces are put before the opening parenthesis in the rest of the codebase.